### PR TITLE
Portfolio:first milestone Project Pull Request

### PR DESCRIPTION
**Pull request by Ahsan12356**
This is the portfolio project. This is the first part of portfolio project which is in the third day of the first week of microverse journey. We have to replicate our favourite template given from the figma templates. In this project we only have replicate header section and headline section of the figma template.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            <div class="time"><p>9:41</p></div>
+            <div class="status_icons">
+                <img src="images/cellular-connection.png" alt="" class="signals">
+                <img src="images/wifi.png" alt="" class="wifi">
+                <img src="images/battery.png" alt="" class="battery">
+            </div>
+            
+        </div>
+        <div class="tool_bar">
+            <p>My Logo</p>
```
Comment (vechicin) [inline index.html:22]: This should be an `<a>` tag, please fix it.

Screen Shot 2022-06-30 at 7 26 39

File: index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            <div><img src="images/icon-menu.png" alt=""></div>
```
Comment (vechicin) [inline index.html:23]: This content should be wrapped in a `<nav>` tag, please fix it.

Screen Shot 2022-06-30 at 7 27 50

File: style.css [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  line-height: 20px;
+
+  /* identical to box height, or 111% */
+
+  align-self: center;
+
+  /* Primary/B400 */
+
+  color: #6070ff;
+}
```
Comment (vechicin) [inline style.css:49]: The logo's left margin doesn't quite match that of the template design, please fix it.

Screen Shot 2022-06-30 at 7 30 54

File: style.css [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+  align-self: center;
+  text-align: center;
+  cursor: pointer;
+}
+
+.headline_sect {
+  background-image: url("images/header-shapes-mobile.png"), url("images/header-background-mobile.png");
+  background-repeat: no-repeat;
+  border: 1px solid #fff;
+}
```
Comment (vechicin) [inline style.css:63]: Your design is missing the `border-radius` in the bottom left corner. Please take a closer look at the Figma design and add it.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Ahsan12356 ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good job overall matching the template design

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by vechicin** [comment]
https://github.com/Ahsan12356/portfolio/blob/c79070c841ae71fba2b53e54e3a8b1868072368f/.github/.workflows/linters.yml#L1

Your linters are not set up properly, please change the `.workflows` name to just `workflows`.

Additionally, when I ran the `npm install command` locally, I got this error in my console

Screen Shot 2022-06-30 at 7 39 03

Please make sure your linters are installed properly.

**Comment by vechicin** [comment]
https://github.com/Ahsan12356/portfolio/blob/c79070c841ae71fba2b53e54e3a8b1868072368f/README.md?plain=1#L1

Please follow the Microverse README template.

File: style.css (hunk 3 above)
Comment (Ahsan12356) [reply style.css:49]: Its exactly the same **12 px**

File: index.html (hunk 1 above)
Comment (Ahsan12356) [reply index.html:22]: ok sure

File: index.html (hunk 2 above)
Comment (Ahsan12356) [reply index.html:23]: ok sure

**Pull request closed without merging**